ambry-tools/src/main/java/com/github/ambry/clustermap/HelixClusterWideAggregationTool.java

```diff
@@ -108,6 +108,10 @@
     @Default("false")
     final boolean deleteSpecifiedWorkflow;
 
+    @Config("is.mysql")
+    @Default("false")
+    final boolean isMysql;
```

Comment:
Time to remove Helix from the class name?  (Unless that will break something.)

Reply:
Helix in the name is fine. Although we start to read the container storage reports from mysql, but this tool will instruct helix to execute these tasks.